src/alfasim_sdk/result_reader/aggregator.py

```diff
@@ -1433,3 +1444,87 @@ def merge_metadata(a_metadata, b_metadata, *, source, update=(), min_=(), max_=(
         result_directory=r_a.result_directory,
         app_version_info=app_version_info,
     )
+
+
+def open_global_sensitivity_analysis_result_file(results_directory: Path) -> h5py.File:
+    """
+    Open a global sensitivity analysis result file.
+    """
+
+    h5py_file = h5py.File
+    if h5py.version.version_tuple[:2] >= (3, 5):
+        h5py_file = functools.partial(h5py.File, locking=RESULT_FILE_LOCKING_MODE)
+    filename = results_directory / "uq_result"
+    return _open_result_file(filename, h5py_file)
+
+
+def get_empty_global_sensitivity_analysis_result_metadata(
+    result_directory: Path,
+) -> UQResultMetadata:
+    """
+    Obtain an empty global sensitivity analysis meta-data.
+    """
+
+    return UQResultMetadata(
+        global_gsa_coefficients={}, result_directory=result_directory
+    )
```

Comment:
- [ ] Instead of this function, create a classmethod `UQResultMetadata.empty()`.